### ISSUE TITLE
Modules do not currently connect

### ISSUE THREAD

**Issue by Chrinkus** [lean-alchemy/shootout#6]
Current code is separated into modules but do not link as modules yet. ES6 includes native support for modules but is not yet finalized. Incorporate ES6 modules using whichever syntax is accepted by Babel.

**Comment by Chrinkus** [comment]
My fork has a working version of the game using ES6 and Babel.

$ babel-node shootout/src/game/tempGameMgr.js

Once this is merged into LA-master issue will be closed.

**Comment by davidwesst** [comment]
Make a pull request to the master branch, with the code. We can reference this issue in the pull request comment and then close it.
